fuzzers/lafintel/builder.Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG parent_image=gcr.io/fuzzbench/base-builder
+FROM $parent_image
+
+# Need Clang/LLVM 3.8.
+RUN apt-get update -y && \
+    apt-get -y install llvm-3.8 \
+    clang-3.8 \
+    libstdc++-5-dev \
+    wget
+
+# Download AFL and compile using default compiler.
+# We need afl-2.26b
+RUN wget https://lcamtuf.coredump.cx/afl/releases/afl-2.26b.tgz -O /afl-2.26b.tgz && \
```

Comment:
This needs to come from google github repo.

Reply:
Not possible since 2.26b isn't something that can be checked out from the repo.

Reply:
yes that's the reason. I could not find it in the google repo.